src/api/learningpathApi.ts

```diff
@@ -28,6 +28,7 @@ export async function fetchLearningpaths(learningpathIds: string[], context: Con
       return {
         id: learningpath.id,
         title: learningpath.title.title,
+        htmlTitle: learningpath.title.title,
```

Comment:
Har vi støtte for html-tittel i stier?

Reply:
Nei, men siden eg utvida Meta-typen måtte eg legge det til her også.